src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -4699,4 +4663,79 @@
     <value>Rotate Through Carry Circular Shift selected</value>
     <comment>Label for a radio button that toggles rotate circular with carry behavior for the shift operations.</comment>
   </data>
-</root>
+  <data name="SettingsButton.Content" xml:space="preserve">
+    <value>Settings</value>
+    <comment>The text that shows in the dropdown navigation control to open the Settings page.</comment>
+  </data>
+  <data name="SettingsButton.AccessKey" xml:space="preserve">
```

Comment:
I'm not sure we want to leave the access key as AB. @grochocki, can you confirm if this is the correct key combination to launch settings?

Reply:
@grochocki Should it be "SE" instead?